src/decode.h

```diff
@@ -995,6 +995,7 @@ void DecodeUnregisterCounters(void);
  * Libpcap on at least OpenBSD returns 101 as datalink type for RAW pcaps though. */
 #define LINKTYPE_RAW2        101
 #define LINKTYPE_IPV4        228
+#define LINKTYPE_IPV6        229
```

Comment:
Where did you find this ? Wireshark code ? Some IANA registration ?

Reply:
```
/usr/include/pcap/dlt.h:#define DLT_IPV6		229
```
btw a follow up task could be to do string representation of the datalink. We know of at least one OS difference:
```
/* taken from pcap's bpf.h */
  #ifndef DLT_RAW
  #ifdef __OpenBSD__
  #define DLT_RAW     14  /* raw IP */
  #else
  #define DLT_RAW     12  /* raw IP */
  #endif
  #endif
  ```